### DISCUSSION TITLE
About anatomy_size

### DISCUSSION THREAD

**Discussion by Wlllllllly** [Project-MONAI/tutorials#1868] [General]
Hi, I am curious about the all_anatomy_size_condtions.json of the MAISI inference. How do you determine the JSON file? Or what are the 10 attributes of each organ_size?

**Comment by Can-Zhao** [comment]
@guopengf

**Comment by guopengf** [comment]
@Wlllllllly Sorry for the late response! ``all_anatomy_size_condtions.json`` is created from real data serving as the reference during inference, since the size of a single organ usually is also correlated to the surrounding organs.

Those 10 attributes have a similar format as ``controllable_anatomy_size`` in the inference parameters. It is a list of controllable anatomy and its size scale (0--1). E.g., if set to [["liver", 0.5],["hepatic tumor", 0.3]], the generated image will contain a liver that has a median size, with the size around 50% percentile, and a hepatic tumor that is relatively small, with around 30% percentile. In addition, if the size scale is set to -1, it indicates that the organ does not exist.
